powerapps-docs/developer/model-driven-apps/clientapi/create-app-side-panes.md

```diff
@@ -91,6 +91,28 @@ Xrm.App.sidePanes.createPane({
 > [!div class="mx-imgBorder"] 
 > ![Screenshot showing an Active Reservations list opened to an individual record.](../media/app-side-panes-opening-record.png "Open record")
 
+### Showing a web resource
+
+This example shows how to display a webresource in the side pane. A custom html page is opened in a side pane where the default header is visible & close button is hidden.
+
+```javascript
+Xrm.App.sidePanes.createPane({
+                        title: "Registration Page",
+                        imageSrc: "WebResources/msfp_SurveyIcon_32",
+                        paneId: "RegistrationPage",
+                        canClose: false
+                    }).then((pane) => {
+                        //navigate to webresource
+                        pane.navigate({
+                            pageType: "webresource",
+                            webresourceName: "new_RegistrationPage",
+                        })
+                    });
+```
+> [!div class="mx-imgBorder"] 
+> ![Screenshot showing a webresource opened]<img width="291" alt="app-side-panes-opening-webresource" src="https://github.com/TheSarcasticGuy/powerapps-docs/assets/37799232/fbdc3fed-18f3-4be6-999f-47d1a032339c">
```

Comment:
@TheSarcasticGuy 

Thanks for your contribution.

I'm afraid we can't have links to external sites for images.
Can you place the image file in this folder: `powerapps-docs/developer/model-driven-apps/clientapi/media` and then include a relative link to it?